test/csuite/timestamp_abort/main.c

```diff
@@ -1653,6 +1707,7 @@ main(int argc, char *argv[])
 
     backup_full_interval = 4;
     backup_granularity_kb = 1024;
+    backup_test_force_stop = false;
```

Comment:
Maybe this doesn't need its own command-line argument, but I wouldn't mind keeping it is a flag that is not controllable via arguments (akin to `backup_verify_immediately`). In this way, we can easily turn this off (with the appropriate FIXME comment) and commit this change, and then turn it back on later. Otherwise it would be hard to merge this PR with the force_stop testing off and without the compiler complaining about unused functions.

Reply:
I like that idea. Take the arg out, but keep as a test setting. Make sure all of the boolean internal settings are declared somewhere together and set together so it is obvious where the one place to change is when debugging.